tests/apple-exporter/components/test-class-date.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Publish to Apple News tests: Date_Test class
+ *
+ * @package Apple_News
+ * @subpackage Tests
+ */
+
+/**
+ * A class to test the behavior of the Apple_Exporter\Components\Date class.
+ *
+ * @package Apple_News
+ * @subpackage Tests
+ */
+class Date_Test extends Apple_News_Testcase {
+
+	/**
+	 * Tests the render method for the component.
+	 */
+	public function test_render() {
+		$this->set_theme_settings( [ 'meta_component_order' => [ 'date' ] ] );
+
+		// Create a test post and get JSON for it.
+		$post_id = self::factory()->post->create( [ 'post_date_gmt' => '1970-01-01 12:00:00' ] );
```

Comment:
Because this evaluates to timestamp `0` and a failed date creation _might_ be cast as a `0`, to make absolutely sure this is working properly, I'd recommend modifying the time slightly:
```suggestion
		$post_id = self::factory()->post->create( [ 'post_date_gmt' => '1970-01-01 12:34:56' ] );
```